packages/sources/finage-test/src/endpoint/http/eod.ts

```diff
@@ -0,0 +1,95 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { customSettings } from '../../config'
+
+export const inputParameters = {
+  base: {
+    aliases: ['from', 'coin'],
```

Comment:
I think in v2 the aliases for eod should be
```suggestion
    aliases: ['from', 'symbol'],
```